SonarQube.TeamBuild.PreProcessor/SonarLintAnalyzerProvider.cs

```diff
@@ -1,179 +0,0 @@
-﻿//-----------------------------------------------------------------------
-// <copyright file="SonarLintAnalyzerProvider.cs" company="SonarSource SA and Microsoft Corporation">
```

Comment:
The SonarLintAnalyzerProvider and tests are no longer required. They are completely replaced by the new RoslynAnalyzerProvider.